GoogleForCreators/web-stories-wp
# Workspace: New Safe Zone Design & Toggle

## Summary

This PR implements a new tick based safe zone, a toggle button in the carousel menu for the ticks, and removes the previous iteration of the danger/safe zone.

## Relevant Technical Choices

I chose to implement the tick marks with top and bottom borders using `:before` and `:after` pseudo elements on the `PageAreaSafeZone` component which are optionally rendered depending on a new value in the `CanvasContext` called `showSafeZone`.


## User-facing changes
This introduces several user-facing change.
1. Removes the danger zone gradient on the canvas
![danger-zone-gradient](https://user-images.githubusercontent.com/2755722/84415468-d0a94680-abe0-11ea-8f3d-53bfbc2c4eeb.png)

2. Adds safe zone tick marks on the sides of the canvas
![safe-zone-ticks](https://user-images.githubusercontent.com/2755722/84415545-ea4a8e00-abe0-11ea-82c5-b1e795e6b5e7.png)

3. Adds a safe zone button to toggle the tick marks to the carousel menu
![toggle-button](https://user-images.githubusercontent.com/2755722/84415583-f6365000-abe0-11ea-9dec-6a309efc08f9.png)



## Testing Instructions
1. Launch into the workspace. The tick marks should be visible by default and the gradient danger zone should be gone.
2. Click the Safe Mode button in the Carousel menu.  This should remove the tick marks and button background.

![2096-safe-zone](https://user-images.githubusercontent.com/2755722/84433533-69998b00-abfc-11ea-8737-5adba45dc136.gif)



#2096
